smoke-tests/spec/ingress_controller_spec.rb

```diff
@@ -3,7 +3,7 @@
 describe "ingress controllers", speed: "fast" do
   context "default" do
     # This needs to be any url of an ingress on the default controller,
-    let(:url) { "https://how-out-of-date-are-we.apps.live-1.cloud-platform.service.justice.gov.uk/dashboard" }
+    let(:url) { "https://reports.cloud-platform.service.justice.gov.uk/dashboard" }
```

Comment:
This won't work on non-live-1 clusters. Ignoring the fact that we will eventually move `reports.cloud.....` onto the new cluster `live`.

Reply:
Perhaps we could use the bootstrap helloworld app?

Reply:
Good point, this test is weird - it will not fail, but will not test the test cluster either; it just pings 1 live and 1 manager address

Reply:
helloworld app +interpolation would fail on live because. it doesn't use the 'apps.cluster' hostname